### PR TITLE
chore(partners): remove sfox and shapeshift

### PR DESCRIPTION
**Pull request by mpfluger**

### CODE REVIEW

File: prod/wallet-options-v4.json [hunk 1]
```diff
@@ -298,56 +298,6 @@
...
-          "TX",
-          "UT",
-          "VA",
-          "WI",
-          "WV",
-          "WY"
-        ],
-        "config": {
-          "production": true,
-          "apiKey": "f31614a7-5074-49f2-8c2a-bfb8e55de2bd",
```
Comment (schnogz) [inline prod/wallet-options-v4.json]: @plondon confirming we dont need this API key to retrieve historical SFOX orders that we may still show in the UI

Comment (plondon) [reply prod/wallet-options-v4.json]: Good catch, seems we do.

https://github.com/blockchain/bitcoin-sfox-client/blob/master/src/api.js#L54

**Comment by akly** [comment]
We should also ensure any API keys due to be removed from our repo are actually invalidated on third-party side (e.g. ShapeShift)  so that they _cannot_ be used even if someone has kept a copy.

**Comment by mpfluger** [comment]
Turns out both are still used for historical reasons to be able to fetch old trades.

**Pull request closed without merging**